presto-verifier/pom.xml

```diff
@@ -108,6 +108,12 @@
             <artifactId>joda-time</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-spi</artifactId>
+            <scope>provided</scope>
```

Comment:
This shouldn't be "provided" it should just be a normal (compile) dependency. Here are some details about what the different scopes are used for: https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope

The reason that most of the connectors set it as provided, is because the presto server provides it at runtime